### ISSUE TITLE
Regression: Battlefield 1 crashes with 0.80, works fine with 0.72

### ISSUE THREAD

**Issue by BrianValente** [doitsujin/dxvk#669]
With the latest version of DXVK at the moment, 0.80, the game crashes while loading. With 0.72 works fine. I tried disabling D3D 11.1 with a DXVK config file and a Battlefield 1 config file but still crashes.

![screenshot_20180925_024235](https://user-images.githubusercontent.com/3992081/45995130-29cbf200-c06d-11e8-8129-cfbcb85694f6.png)

### Software information
Battlefield 1

### System information
- GPU: NVIDIA GTX 970
- Driver: 396.54.05
- Wine version: wine-3.16 (Staging)
- DXVK version: 0.80

### Apitrace file(s)
- Can't make apitrace. bf1.exe closes immediately and it's opened again by Origin.exe.

### Log files
- d3d11.log: [bf1_d3d11.log](https://github.com/doitsujin/dxvk/files/2413935/bf1_d3d11.log)
- dxgi.log: [bf1_dxgi.log](https://github.com/doitsujin/dxvk/files/2413934/bf1_dxgi.log)

**Comment by doitsujin** [comment]
Can you bisect to the exact commit which introduces the issue?

**Comment by BrianValente** [comment]
I'm very sorry for losing your time. Looks like the problem was after I changed my screen resolution. I deleted the game save files, reinstalled, and everything was working again.
Makes sense with FindClosestMatchingMode, looks like something went wrong when looking for the previous resolution I was using. I don't know if it's a problem in DXVK or Xorg, but it's not that important and easy to solve.

Sorry and thank you.
